CondCore/Utilities/python/conddblib.py

```diff
@@ -5,7 +5,7 @@
 __copyright__ = 'Copyright 2013, CERN'
 __credits__ = ['Giacomo Govi', 'Miguel Ojeda', 'Andreas Pfeiffer']
 __license__ = 'Unknown'
-__maintainer__ = 'Giacomo Govi'
+__maintainer__ = 'Miguel Ojeda'
```

Comment:
@franzoni this seems to revert #17851 and #17278 from @ggovi. Is this intentional?